prime-router/docs/schema_documentation/ipatientcare-reddyfmc-la-covid-19.md

```diff
@@ -445,11 +445,14 @@ Code | Display
 ---- | -------
 Holly|Holly Delatte NP
 De Anna|De Anna Dark FNP-C
+DeAnna|DeAnna Dark FNP-C
 Nagaratna|Nagaratna Reddy MD.
 Cassandra|Cassandra Hill-Selders NP
 Crystal|Crystal Rivet NP
 Darrell|Darrell Davis NP
 Wanda|Wanda Jefferson Wilson FNP-C
+Wanda|Wanda Jefferson FNP-C
```

Comment:
Just curious: is it okay to have two "Display" values for the same "Code"?

Reply:
@cwinters-usds let me double check with @rhood23699